src/lib/deploy.js

```diff
@@ -42,6 +42,16 @@ export async function deploy({ alias, yes }) {
   const latestCliVersion = await getLatestCliVersion();
   const installedCliVersion = getInstalledCliVersion();
 
+  if (!installedCliVersion) {
+    log(
+      chalk.red(
+        `Failed to detect the installed zkapp-cli version. This might be possible if you are using Volta to manage your Node versions.`
```

Comment:
```suggestion
        `Failed to detect the installed zkapp-cli version. This might be possible if you are using Volta or something similar to manage your Node versions.`
```

Reply:
Thanks for adding this @satyambnsal. I suggest we make the error message a little more general.